src/components/bottom-sheet/use-bottom-sheet.tsx

```diff
@@ -0,0 +1,217 @@
+import React, {
+  ReactNode,
+  RefObject,
+  useCallback,
+  useRef,
+  useState,
+} from 'react';
+import {
+  AccessibilityInfo,
+  Animated,
+  findNodeHandle,
+  LayoutChangeEvent,
+  TouchableWithoutFeedback,
+  View,
+} from 'react-native';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+import {BottomSheetTexts, useTranslation} from '@atb/translations';
+
+const Backdrop = ({isOpen}: {isOpen: boolean}) => {
+  const styles = useStyles();
+  return (
+    <View
+      style={{
+        ...styles.backdrop,
+        opacity: isOpen ? 0.2 : 0,
+      }}
+      pointerEvents={'box-none'}
+    />
+  );
```

Comment:
```suggestion
  const styles = useStyles();
  const fadeAnim = useRef(new Animated.Value(isOpen ? 0.5 : 0)).current;

  React.useEffect(() => {
    Animated.timing(fadeAnim, {
      toValue: isOpen ? 0.5 : 0,
      duration: 200,
      easing: Easing.inOut(Easing.ease),
      useNativeDriver: true,
    }).start();
  }, [isOpen, fadeAnim]);

  return (
    <Animated.View
      style={[{opacity: fadeAnim}, styles.backdrop]}
      pointerEvents={'box-none'}
    />
  );
```

Reply:
Fixed by reusing the `animatedOffset` for the bottom sheet itself. Still using `0.2` as opacity as that is the value used in Figma.